1-org/envs/shared/README.md

```diff
@@ -29,13 +29,25 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
 1. Merge changes to prod branch with `git checkout -b prod` and `git push origin prod`
     1. Review the apply output in your cloud build project https://console.cloud.google.com/cloud-build/builds?project=YOUR_CLOUD_BUILD_PROJECT_ID
 
+
 ### Run terraform locally
-1. Change into 1-org/envs/shared/ folder.
+1. Change into 1-org folder.
+1. Run `cp ../build/tf-wrapper.sh .`
+1. Run `chmod 755 ./tf-wrapper.sh`
 1. Rename terraform.example.tfvars to terraform.tfvars and update the file with values from your environment and bootstrap.
-1. Update backend.tf with your bucket from bootstrap.
-1. Run `terraform init`
-1. Run `terraform plan` and review output.
-1. Run `terraform apply`
+1. Update backend.tf with your bucket from bootstrap. You can run
+```for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/<YOUR-BUCKET-NAME>/' $i; done```.
+You can run `terraform output gcs_bucket_tfstate` in the 0-bootstap folder to obtain the bucket name.
+
+We will now deploy our environment (prod) using this script.
+When using Cloud Build or Jenkins as your CI/CD tool each environment corresponds to a branch is the repository for 1-org step and only the corresponding environment is applied.
+
+1. Run `./tf-wrapper.sh init prod`
+1. Run `./tf-wrapper.sh plan prod` and review output.
```

Comment:
Sorry for late review, have we tested this locally for `shared`?

I would have thought we will run into the same issue as Cloud Build, where it errors and exits after trying to do the plan for `prod`? It might still work, if the plan file has been created - but on both steps it will error out.

If this is correct ^ we may need to actually executed terraform locally, similar to original instructions.